File: viewbindtest/src/main/java/com/example/viewbindtest/MainActivity.java
package com.example.viewbindtest;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.viewbindtest.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {


    ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
//        setContentView(R.layout.activity_main);

        binding = ActivityMainBinding.inflate(getLayoutInflater());
    }
}

File: myapplication/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import androidx.databinding.DataBindingUtil
import com.example.myapplication.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {


    lateinit var binidng: ActivityMainBinding;

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
//        setContentView(R.layout.activity_main)


        var count: Int = 10;
        val count1: Int = 20;

        binidng = DataBindingUtil.setContentView(this, R.layout.activity_main)






    }
}

